### DISCUSSION TITLE
this is a such a proper plugin bro good job. looks so good sir

### DISCUSSION THREAD

**Discussion by paxelpixel** [pineapplegiant/spacedirt-theme#16] [General]
love this sooo much

**Comment by pineapplegiant** [comment]
Ayy thanks friend, it means a lot 😄

**Comment by repl6669** [comment]
Thank you very much for spaceduck, it looks and feels 🔥. 

One silly question - please how did you make the NERDTree (is it NT?) look so good on the first screenshot in readme? I tinkered with it a bit, but could not get it anywhere near. I assume you used Devicons, but what other configuration did you use please?

Thanks again.

**Reply by pineapplegiant** [reply]
I'm using the [Fern](https://github.com/lambdalisue/fern.vim) plugin for my filetree along with a few other fern supplemental plugins to get it working with icons. I recommend it over NT as it's a bit more performant, but I'll take a look at configuring the colors for NT later no problem.

```viml
    Plug 'lambdalisue/fern.vim'                   "Filetree alternative to Nerdtree
    Plug 'lambdalisue/fern-git-status.vim'        "Add git dirty status to filetree
    Plug 'lambdalisue/fern-hijack.vim'            "Makes fern the default file explorer instead of netrw
    Plug 'lambdalisue/nerdfont.vim'               "I forgot what this does truthfully
    Plug 'lambdalisue/fern-renderer-nerdfont.vim' "Basically vim-devicons, needed for icons
    Plug 'lambdalisue/glyph-palette.vim'          "Adds personal colors (not needed, if wanting the default colors in the screenshot)
```

**Reply by repl6669** [reply]
Oh nice, thank you very much for sharing, did not know about it.

**Reply by pineapplegiant** [reply]
Let me know if you want any more help with the customization?

I installed [Devicons](https://github.com/ryanoasis/vim-devicons) along with [Nerdtree syntax highlighting](https://github.com/tiagofumo/vim-nerdtree-syntax-highlight) to play with it–and I don't think the default colors are too bad on the icons. A screenshot is below to show how they look, I'm using [Sauce Code Pro Font Mono](https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/SourceCodePro) for my font. (If you're on mac you can install it with [brew](https://brew.sh/) and the command is (`brew install --cask font-sauce-code-pro-nerd-font`). My [dotfiles](https://github.com/pineapplegiant/dotfiles/tree/supreme-overlord/.config/nvim) aren't pretty, but you can always look at them if you're curious to how I configure everything.

The default plugins used in the screenshot: 
```viml
    Plug 'preservim/nerdtree'
    Plug 'tiagofumo/vim-nerdtree-syntax-highlight'
    Plug 'ryanoasis/vim-devicons'
```


![Screen Shot 2021-03-10 at 6 48 47 PM](https://user-images.githubusercontent.com/32819563/110723420-40c91500-81d1-11eb-83fd-99d01a7878c2.png)

**Reply by repl6669** [reply]
Thank you very much for all your effort. I find Fern to work beautifully, so I ditched NERDTree all together.

I patched my own "nerd font", because I use Dank mono and now it has everything it needs.

I only found one tiny flaw (which may be a problem on my end). I think that there may be one color missing for PHP. I attach 2 screenshots comparing Spaceduck with Material. I also use vim-polyglot as recommended.

![Screenshot 2021-03-15 at 23 24 34](https://user-images.githubusercontent.com/5587309/111229834-ed96fe00-85e6-11eb-8f95-e6b653d54063.png)

![Screenshot 2021-03-15 at 23 23 14](https://user-images.githubusercontent.com/5587309/111229841-f091ee80-85e6-11eb-843b-944ef8ce19d7.png)

As you can see, Material has also syntax highlighting for strings. For example `'return_url'`.

Otherwise I am 100% happy with Spaceduck. Thank you again.

**Reply by pineapplegiant** [reply]
Great find! I have strings and identifiers as the cyan color currently–But I don't really like how it looks either. I will come up with better colors for php and push the changes this week.

If you find any other things that look off, feel free to post here. I'll let you know when I make those changes 🚀 🦆

**Reply by pineapplegiant** [reply]
I just made this change. I added a bit more green and moved the pink to where the blue was. Not sure if I'm in love with it, but I used the Material theme as inspiration to add the Yellow.
<img width="1361" alt="Screen Shot 2021-03-16 at 10 36 57 PM" src="https://user-images.githubusercontent.com/32819563/111415727-1f23ce00-86a8-11eb-8aef-1d3b3565b6dc.png">

**Reply by repl6669** [reply]
I think it looks much nicer. 😍
